<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBitMessageMailBoxDialog</class>
 <widget class="QDialog" name="NewBitMessageMailBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Bitmessage mailbox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mailBoxLabel"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mailBoxType">
       <item>
        <property name="text">
         <string>BitMessage (regular)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="useAsDefault">
       <property name="text">
        <string>Use as default mailbox</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="iconCombo"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
